saige_assoc.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python3
+# pylint: disable=too-many-arguments
+
+"""
+Hail Batch workflow to perform association tests using SAIGE-QTL
+
+- given all input files already generated
+    - pheno_cov filename, genotypes with annotations
+- builds saige commands (just add to str)
+- run saige commands (execute Rscript from command line)
+- aggregate & summarise results (not yet)
+
+"""
+
+import click
+
+import hailtop.batch as hb
+from cpg_utils.hail_batch import get_config, remote_tmpdir
```

Comment:
```suggestion
from cpg_utils.config import get_config
from cpg_utils.hail_batch import get_batch
```

Technically `from cpg_utils.hail_batch import get_config` works, but it's a chained import. `get_config` is actually in `cpg_utils.config`, but `cpg_utils.hail_batch` [imports it](https://github.com/populationgenomics/cpg-utils/blob/main/cpg_utils/hail_batch.py#L21). By importing the way you are here, this is prone to failing in the future if the cpg-utils code is rearranged future.

The `get_batch` wrapper [here](https://github.com/populationgenomics/cpg-utils/blob/main/cpg_utils/hail_batch.py#L44) should be used instead of directly invoking a batch from parts as you are here (generating the tempdir/service backend etc.)